Sustainsys.Saml2/SAML2P/Saml2PSecurityTokenHandler.cs

```diff
@@ -17,10 +17,15 @@ namespace Sustainsys.Saml2.Saml2P
 	/// could be handled at transport level.
 	/// </summary>
 	public class Saml2PSecurityTokenHandler : Saml2SecurityTokenHandler
-    {
-		public Saml2PSecurityTokenHandler()
+	{
+		public Saml2PSecurityTokenHandler(SPOptions spOptions)
```

Comment:
This is a breaking change as the Saml2PSecurityTokenHandler is a public class.

I'd prefer if the old param-less ctor is kept side-by-side and thus the options has to be optional. Which means optional also for the Saml2PSerializer...

What do you think?